webpack.config.js

```diff
@@ -31,35 +32,45 @@ module.exports = {
   },
   plugins: [
     new HtmlWebpackPlugin({
-      template: './src/options.html',
+      template: "./src/options.html",
       filename: "options.html",
-      chunks: ['js/options']
+      chunks: ["js/options"],
     }),
     new HtmlWebpackPlugin({
-      template: './src/postHandler.html',
+      template: "./src/postHandler.html",
       filename: "postHandler.html",
-      chunks: ['js/postHandler']
+      chunks: ["js/postHandler"],
     }),
     new HtmlWebpackPlugin({
-      template: './src/migration.html',
+      template: "./src/migration.html",
       filename: "migration.html",
-      chunks: ['js/migration']
+      chunks: ["js/migration"],
     }),
     new CopyWebpackPlugin({
       patterns: [
         {
           // Copy assets from 'src/' and settings file to 'dist/' folder.
           from: path.join(__dirname, "src/"),
           globOptions: {
-            ignore: ["**/*.+(css|js)", '**/migration.html', '**/options.html', '**/postHandler.html'],
+            ignore: [
+              "**/*.+(css|js)",
+              "**/migration.html",
+              "**/options.html",
+              "**/postHandler.html",
+            ],
           },
         },
         {
           from: "./settings.json",
         },
       ],
     }),
-    new bomPlugin(true)
+    new bomPlugin(true),
+    new webpack.DefinePlugin({
+      "process.env.MIXPANEL_TOKEN": JSON.stringify(
+        "bf3d97d87d23b7e6cc636c5159c1e90d"
```

Comment:
I forgot to mention this, but we shouldn't hardcode the Mixpanel token: instead, we should use [files with environment variables](https://webpack.js.org/guides/environment-variables/), like `prod.env` and `dev.env`.